src/common/options/rgw.yaml.in

```diff
@@ -4498,3 +4498,28 @@ options:
   default: false
   services:
   - rgw
+- name: rgw_ratelimit_accumulation_interval_seconds
+  type: uint
+  level: advanced
+  desc: Time window for rate limiting in seconds
+  long_desc: This option sets the time window for rate limiting in seconds.
+    Requests that exceed the configured rate limits within this time window will be rejected.
+    The default is 1 minute token bucket.
+  default: 60
```

Comment:
@mattbenjamin re:
> @mkogan1 independent of this change, do you think it would be viable to make the rate limit accumulation interval configurable?

added a conf: `rgw_ratelimit_accumulation_interval_seconds`